install/kubernetes/helm/istio/charts/ingress/templates/service.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
     istio: ingress
   annotations:
     {{- range $key, $val := .Values.service.annotations }}
```

Comment:
Can you swipe the whole chart and fix all?

Reply:
added quote option to other service annotations :)